examples/paths.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+default_cf_mapping_files = [os.path.dirname(__file__) + '/cloudformation/cloudformation_map.yaml',
```

Comment:
I prefer to don't do this change right now. It will require changes not just in the code, but also in the deployment process and currently both of them are not in a mature state. I think it will be better to wait until we investigate a bit more about the real mapping files that we are going to use and to test a bit more the deployment